scientist-softserv/palni-palci
# Oai for cdls

# Story
- add the 3 new properties from the CDL worktype to the OAI feed. 

# Related
#461 

# Expected Behavior Before Changes
- users would not see `contributing_library`, `library_catalog_identifier`, or `chronology_note` in the oai feed

# Expected Behavior After Changes
- users will see `contributing_library`, `library_catalog_identifier`, `chronology_note` for CDL works with data for these properties

# Screenshots / Video

<details>
<summary>3 additional properties in the OAI Feed</summary>
https://github.com/scientist-softserv/palni-palci/pull/new/oai-for-cdls
</details>

# Testing Instructions
1. Create a CDL work on the test tenant (or whichever tenant you prefer). Make sure to fill out the `contributing_library`, `library_catalog_identifier`, and`chronology_note` properties
2. Visit https://test.palni-palci-staging.notch8.cloud/catalog/oai?verb=ListRecords&metadataPrefix=oai_hyku (or change "test" to your tenant's name if necessary)
3. Search for your work and confirm you see those 3 properties.
